api/tests/test_views.py

```diff
@@ -1,64 +1,154 @@
 import pytest
+import json
 
+from django.test import TestCase
+from django.utils import timezone
+from rest_framework.test import RequestsClient
 
-def test_fire():
-    """
-    Employees related test
-    """
-    pass
+from ..factories import EmployeeFactory, IncomeFactory, FutureEventFactory, TodaysEventsFactory, PastEventFactory
 
+SERVER_ADDRESS = 'http://127.0.0.1:8000'
+CLIENT = RequestsClient()
```

Comment:
Use `from rest_framework.test import APIRequestFactory`.

https://www.django-rest-framework.org/api-guide/testing/#creating-test-requests

Reply:
How about using `client` fixture provided by `pytest-django` package? 
https://pytest-django.readthedocs.io/en/latest/helpers.html#client-django-test-client

As it's a fixture already it does not have to be initialised inside the module, just put it in the arguments list in each test.